libcontainer/dmz/_dmz.c

```diff
@@ -11,5 +13,17 @@ int main(int argc, char **argv)
 {
 	if (argc < 1)
 		return 127;
-	return execve(argv[0], argv, environ);
+	int r = execve(argv[0], argv, environ);
+	if (r) {
+		char *prefix = "exec ";
+		int err_len = strlen(prefix) + strlen(argv[0]) + 1;
+		char err[err_len];
```

Comment:
Do not use VFAs. They are fundamentally broken.

Reply:
@cyphar what do you suggest? A malloc here?

Reply:
A fixed-size buffer would be simpler. See my suggested patch (though `strlcat` doesn't work even though it's defined in `nolibc/string.h`...)